python/tests/test_aac/context/test_language_context.py

```diff
@@ -107,6 +109,53 @@ def test_get_values_by_field_chain(self):
             values = context.get_values_by_field_chain(val)
             self.assertGreater(len(values), 1)
 
+    def test_create_aac_object(self):
+        context = LanguageContext()
+        object = context.create_aac_object("Schema", {})
+        self.assertEqual(str(type(object)), "<class 'aac.lang.Schema'>")
+
+    def test_create_aac_object_incorrect_type(self):
+        with self.assertRaises(AttributeError):
+            context = LanguageContext()
+            object = context.create_aac_object("Schema", "Attribute")  # noqa: F841
+
+    # Something is going wrong with this method, it is failing to create the class. Putting a pin in it.
+    # def test_create_aac_enum(self):
+    #     context = LanguageContext()
+    #     enum = context.create_aac_enum("RequirementVerificationMethod", "TEST")
+
+    # def test_create_aac_enum_incorrect_type(self):
+    #   with self.assertRaises(AttributeError):
+    #       context = LanguageContext()
+    #       enum = context.create_aac_enum("RequirementVerificationMethod", 123)
+
+    def test_parse_and_load_non_definition(self):
+        with self.assertRaises(ParserError):
+            context = LanguageContext()
+            context.parse_and_load("definition")
+
+    def test_get_python_type_from_primitive_no_type(self):
+        context = LanguageContext()
+        with self.assertRaises(LanguageError):
+            context.get_python_type_from_primitive("type")
+
+    # Should this method return an error of some kind when it finds no definitions of a name?
```

Comment:
Yes it should

Reply:
Guessing the question came up because you weren't getting a dedicated error message? If so, capture it as a gap. They should all return an internal LanguageError or ParserError.

Reply:
I was wondering, because I wasn't sure if we wanted the `get_definitions_by_name()` method to act more like a search function that returns an empty list if nothing is found (Which is what it does now), or if we always want it to find something.